playbooks/robusta_playbooks/cpu_throttling.py

```diff
@@ -1,4 +1,6 @@
-from robusta.api import *
+import logging
```

Comment:
I see that you had to manually import logging when you removed the wildcard, 
just double check that you didn't forget to add this import to other places that you replaced the wild card that need it

Reply:
flake8 and pyright actually raise errors if something is not imported or missing - so it should have found all such places

Reply:
amazing, does it throw in skaffold or only when pushed to git?

Reply:
No, it only shows when you run flake8 or pyright - no runtime errors or runtime checks